.github/actions/templates/avm-validateModuleDeployment/action.yml

```
@@ -55,14 +55,47 @@ inputs:
   customLocation:
     description: "Custom location overwrite, if needed"
     required: false
+  e2eIgnore:
+    description: "A test case will not be deployed if an .e2eignore file exists"
+    required: false
```

I'd suggest to either set a false default value or make the input required
```suggestion
    default: "false"
    required: false
```